cr8/fake_providers.py

```diff
@@ -28,6 +29,36 @@ def auto_inc(fake):
     return Counter(manager.Value('i', 0), manager.Lock())
 
 
+def _dest_point(point, distance, bearing, radius):
+    # calculation taken from
+    # https://cdn.rawgit.com/chrisveness/geodesy/v1.1.2/latlon-spherical.js
+    # https://www.movable-type.co.uk/scripts/latlong.html
+
+    δ = distance / radius  # angular distance in rad
```

Comment:
:scream:

Reply:
would you rather use descriptive variable names?

Reply:
No I think it's okay since the formulas use these symbols as well =)